### DISCUSSION TITLE
iOS Chrome v113 now supports fixed scaling when using Auto USDZ

### DISCUSSION THREAD

**Discussion by milesgreen** [google/model-viewer#4212] [General]
As of iOS Chrome v113 (beta), AR QuickLook parameters can now be passed when opening a USDZ blob. Meaning that when using the ModelViewer Auto USDZ feature in iOS Chrome, the fixed scaling parameter will now be observed. Prior to this, the fixed scaling parameter was ignored when using a USDZ blob, and the user was able to resize the AR model. 

No changes required in ModelViewer. It'll just start working better in iOS Chrome v113 🎉

**Comment by elalish** [comment]
Thanks for sharing!
